browser/src/control/jsdialog/Widget.FormulabarEdit.js

```diff
@@ -172,8 +174,17 @@ function _setSelection(cursorLayer, text, startX, endX, startY, endY) {
 
 	// possible after cursor is added to the DOM
 	if (cursor) {
-		var blockOption = JSDialog._scrollIntoViewBlockOption('nearest');
-		cursor.scrollIntoView({behavior: 'smooth', block: blockOption, inline: 'nearest'});
+		if (scrollToCursorTimeout)
+			clearTimeout(scrollToCursorTimeout);
+
+		// put scrol at the end of the task queue so we will not scroll multiple times
```

Comment:
Soo ... we have a nice event for this that doesn't require a timeout:
src/layer/tile/TileLayer.TableOverlay.js:               this._map.on('messagesdone', this._updateTableMarkers, this);
best to slave to that if we can in a follow-up commit.

Reply:
Why is not using a timeout better ? it's more deterministic to rely on messagesdone and reduces the number of interleavings we see / test =)

Reply:
It was intentional as that callback code is independent and result doesn't affect anything else in the app. 

I tried to avoid "mandatory" scroll after processing every messages chunk (in case of multiple very active users, where messages are produced faster than consumed). JS env task is possible to cancel in that case.

messagesdone: |messages|scroll|messages|scroll|messages|scroll|
setTimeout "0":  |messages|~~scroll~~|messages|~~scroll~~|messages|scroll|

But I did quick test with 6 editors emiting events by typing in parallel. I've seen no difference between `messagesdone` and `setTimeout` in profile  so probably it's hard to produce situation when it has impact.

Reply:
followup: https://github.com/CollaboraOnline/online/pull/7615